package.json

```diff
@@ -12,16 +12,15 @@
   ],
   "author": "Julie Ralph <ju.ralph@gmail.com>",
   "dependencies": {
-    "jasmine": "2.4.1",
-    "selenium-webdriver": "3.0.1"
+    "jasmine": "2.4.1"
```

Comment:
We never actually `require('jasmine')` in `index.js`, so we could make this a `devDependency`.  I'm not sure explicitly depending on `jasmine` actually does anything for us.